rental_list.py

```diff
@@ -0,0 +1,29 @@
+import requests
+# import datetime
+
+class RentalList:
+    def __init__(self, url="http://localhost:5000", selected_rental=None):
+        self.url = url
+        self.selected_rental = selected_rental
+
+    def check_out(self, customer_id, video_id):
+        query_params = {
+            "customer_id": customer_id,
+            "video_id": video_id,
+        }
+
+        response = requests.post(self.url+"/rentals/check-out", json=query_params)
```

Comment:
What happens if the check-out route returns a 400 or 404 because there is no available inventory? Just as with check-in, consider how we should handle these 400 level responses.